README.rst

```diff
@@ -40,19 +40,15 @@ Or, to set the primary color to forest green, run::
     # Note: The nested quotes are needed in order to handle the hash (#) correctly.
     tutor config save --set 'INDIGO_PRIMARY_COLOR="#225522"'
 
-Enabling Dark Theme
+Theme Toggle Button
 -------------------
 
-To enable the dark theme for the whole platform, run::
+The theme toggle button is enabled by default when Tutor Indigo is installed. The theme can be switched from light to dark and vice versa. To disable it, run::
 
-    tutor config save --set INDIGO_ENABLE_DARK_THEME=True
+    tutor config save --set INDIGO_THEME_COOKIE_NAME=null
```

Comment:
When the feature is disabled this way, will we default to light or dark theme?

Reply:
If the feature is disabled, then it will be light theme by default.

Reply:
We should not use the cookie name as a toggle feature. We should:

1. remove the  `INDIGO_THEME_COOKIE_NAME` setting.
2. introduce a boolean `INDIGO_ENABLE_DARK_TOGGLE` that defaults to true.